.github/workflows/cd.yml

```
@@ -46,12 +47,19 @@ jobs:
+            echo "OCR_HOST=${{ secrets.OCR_HOST }}" >> .env
+            echo "OCR_PATH=${{ secrets.OCR_PATH }}" >> .env
+            echo "VAPID_PUBLIC=${{ secrets.VAPID_PUBLIC }}" >> .env
+            echo "VAPID_PRIVATE=${{ secrets.VAPID_PRIVATE }}" >> .env            
             echo "NAVER_MAP_CLIENT_ID=${{ secrets.NAVER_MAP_CLIENT_ID }}" >> .env
             echo "NAVER_MAP_CLIENT_SECRET=${{ secrets.NAVER_MAP_CLIENT_SECRET }}" >> .env
             echo "NAVER_DATALAB_CLIENT_ID=${{ secrets.NAVER_DATALAB_CLIENT_ID }}" >> .env
             echo "NAVER_DATALAB_CLIENT_SECRET=${{ secrets.NAVER_DATALAB_CLIENT_SECRET }}" >> .env
             echo "DOCKER_IMAGE=${{ secrets.DOCKER_IMAGE }}" >> .env
-
+            echo "KAKAO_REST_API_KEY=${{ secrets.KAKAO_REST_API_KEY }}" >> .env
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)